src/write.jl

```diff
@@ -572,8 +529,11 @@ function Base.write(io::IO, model::Model{T}) where {T}
         print(io, sense == MOI.MAX_SENSE ? "maximize " : "minimize ")
         F = MOI.get(model, MOI.ObjectiveFunctionType())
         f = MOI.get(model, MOI.ObjectiveFunction{F}())
-        print(io, _to_string(variables, f; include_constant = true))
+        _write_expression(io, predicates, variables, f)
         println(io, ";")
     end
+    for p in predicates
+        println(io, "include \"", p, ".mzn\";")
```

Comment:
Ah. I guess I never tested including these at the end of the file. I just assumed they needed to be before the actual usage.